apps/aecore/src/aec_keys.erl

```diff
@@ -78,6 +79,18 @@ sign(Term) ->
 pubkey() ->
     gen_server:call(?MODULE, pubkey).
 
+-spec wait_for_pubkey() -> {ok, binary()}.
+wait_for_pubkey() ->
+    wait_for_pubkey(1).
+
+wait_for_pubkey(Sleep) ->
+    case pubkey() of
```

Comment:
This wait_for_pubkey may wait forever e.g. if node was started without password.
The code could be simplified by [making `aec_keys` initialization simpler](https://www.pivotaltracker.com/story/show/152483887) hence avoiding need for this method. Though I understand this is not the focus of this PR so no need to change it yet (if I find strong reason to amend this I will add new comment).